### DISCUSSION TITLE
Numerous problems I can't seem to figure out. Also motor starts spinning when not armed.

### DISCUSSION THREAD

**Discussion by DuckSoup88** [iNavFlight/inav#6909] [Q&A] [answered]
I'm having a number of problems getting my Flight Controller to work. I've researched as much as I could, and I'm just out of ideas. Let me detail the problems...

Equipment: 

Mateksys F765-Wing (Flashed to latest firmware)
FrySky X8R receiver
RadioMaster Tx16

The first problem and really my biggest is that my motor will start spinning randomly even when the FC is not armed. The second I create a motor mix and save the changes, it will start spinning at a very low speed. 

I've tried resetting the FC and resetting everything on my transmitter. (No mixes being used on my Tx, or special settings)

When I reset the FC, I calibrate the sensors, create a servo and motor mix, and assign the ARM mode to a switch. These steps alone will result in the motor starting to spin on its own. I have no idea what I'm doing wrong here. 

The second problem is also related to the motor. When I arm the FC, I get a yellow ! in iNAV, and sometimes the motor will spin, sometimes it wont. The pre-arm checks of the FC are all green, nothing showing up as red. I'm not sure what else I'm supposed to check. Sometimes my Tx will say that it's arming, sometimes it won't do anything. 

The other issues are more minor and appear to be related to telemetry, but I won't worry about that stuff just yet.

**Comment by avsaase** [comment, marked as answer]
Have you calibrated your esc?

**Reply by DuckSoup88** [reply]
I have not, I'll do some research on how to calibrate the ESC.

**Reply by avsaase** [reply]
This video explains it quite well: https://www.youtube.com/watch?v=9B6sel9jCF0

**Reply by avsaase** [reply]
About your other problem, the yellow ! is just an indication that the fc is armed so that works as it should. I expect the motor spinning will be solved by calibrating the ESC. The voice callouts from the telemetry script are very unreliable in my experience, so I wouldn't worry about them.

**Reply by DuckSoup88** [reply]
Awesome, thank you for the Youtube video. 

One thing I learned actually, was that most of my problems happened when I have the FC plugged into my PC while also having the battery connected. 

If I just plugged in the battery, I had no other problems.
